file-attribute-caching/src/main/kotlin/com/pkware/file/fileAttributeCaching/FileAttributeCachingPath.kt

```diff
@@ -0,0 +1,209 @@
+package com.pkware.file.fileAttributeCaching
+
+import com.pkware.file.forwarding.ForwardingPath
+import java.nio.file.FileSystem
+import java.nio.file.Path
+import java.nio.file.attribute.BasicFileAttributes
+import java.nio.file.attribute.DosFileAttributes
+import java.nio.file.attribute.PosixFileAttributes
+import java.util.concurrent.TimeUnit
+import java.util.function.Function
+
+// Set the cache preservation duration to 5 seconds as most file operations complete in this time.
+private const val CACHE_PRESERVATION_SECONDS: Long = 5
```

Comment:
Seems like a value that should be externally configurable in some way, perhaps by way of a static method.

Reply:
done but not static